README.md

```diff
@@ -1,2 +1,54 @@
-# service-squared
-A new way for service members to be a part of their commmunity.
+
+# README.md
```

Comment:
This is a nice clean ReadMe with good instructions and images.  It's a wise idea to put something like this in any repo that you make, as it is one of the most readily visible representations of your commitment to your work and its quality.  Well done!